src/sensitivity_algorithms.jl

```diff
@@ -1021,10 +1021,24 @@ like BLAS/LAPACK are used) and this will be the most efficient adjoint implement
 ## Constructor
 
 ```julia
-EnzymeVJP(compile=false)
+EnzymeVJP(;chunksize=12)
 ```
+
+## Keyword Arguments
+
+- `chunksize`: the default chunk size for the temporary variables inside of the vjp's right
+  hand side definition. This is used for compatibility with ODE solves that default to using
+  ForwardDiff.jl for the Jacobian of the stiff ODE solve, such as OrdinaryDiffEq.jl. This
+  should be set to the maximum chunksize that can occur during an integration to preallocate
+  the `DualCaches` for PreallocationTools.jl. It defaults to 12, the maximum default
+  chunksize of ForwardDiff.jl, for safety, but could be decreased if this value is known to
+  be lower to conserve memory. Future optimizations can make this more automatic.
 """
-struct EnzymeVJP <: VJPChoice end
+struct EnzymeVJP <: VJPChoice
+    chunksize::Int
+end
+
+EnzymeVJP(; chunksize = 12) = EnzymeVJP(12)
```

Comment:
Shouldn't it be:
```suggestion
EnzymeVJP(; chunksize = 12) = EnzymeVJP(chunksize)
```
?

Reply:
yes, good catch

Reply:
Fixed on master